estate/__manifest__.py

```diff
@@ -1,15 +1,16 @@
+# noinspection PyStatementEffect
 {
     'name': "Estate",
     'version': '1.0',
     'depends': ['base'],
     'author': "Alessandro (alca)",
-    'application':True,
-    'data':[
+    'application': True,
+    'data': [
```

Comment:
Let's practice git: try to apply the fixes within the commit where the line initially appeared.
You'll most likely need to use `git rebase -i` and `git push --force-with-lease`.